packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-steps.scss

```diff
@@ -0,0 +1,43 @@
+.wallet-steps {
+    width: 100%;
+    height: 100%;
+    display: grid;
+    place-content: center;
+    align-items: center;
```

Comment:
remove `align-items: center;`. It is not needed because Place Content is `center`.